services/management/service_test.go

```diff
@@ -146,5 +147,115 @@ func TestServiceService(t *testing.T) {
 			assert.Nil(t, service)
 			tests.AssertGRPCError(t, status.New(codes.NotFound, `Service with ID "/service_id/00000000-0000-4000-8000-000000000005" not found.`), err)
 		})
+
+		t.Run("RDS", func(t *testing.T) {
+			ctx, s, teardown := setup(t)
+			defer teardown(t)
+
+			node, err := models.CreateNode(s.db.Querier, models.RemoteRDSNodeType, &models.CreateNodeParams{
+				NodeName: "test",
+				Address:  "test-address",
+				Region:   pointer.ToString("test-region"),
+			})
+			require.NoError(t, err)
+
+			service, err := models.AddNewService(s.db.Querier, models.MySQLServiceType, &models.AddDBMSServiceParams{
+				ServiceName: "test-mysql",
+				NodeID:      node.NodeID,
+				Address:     pointer.ToString("127.0.0.1"),
+				Port:        pointer.ToUint16(3306),
+			})
+			require.NoError(t, err)
+
+			pmmAgent, err := models.CreatePMMAgent(s.db.Querier, models.PMMServerNodeID, nil)
+			require.NoError(t, err)
+
+			mysqldExporter, err := models.CreateAgent(s.db.Querier, models.MySQLdExporterType, &models.CreateAgentParams{
+				PMMAgentID: pmmAgent.AgentID,
+				ServiceID:  service.ServiceID,
+				Password:   "password",
+				Username:   "username",
+				// TODO TLS
```

Comment:
todo

Reply:
I can't find it in Jira, but we have TLS support for MonogDB. Most likely we have plans also for MySQL. This section is copy-pasted from existing tests.